nifi-docs/src/main/asciidoc/administration-guide.adoc

```diff
@@ -1720,9 +1720,9 @@ For each Node, the minimum properties to configure are as follows:
    thread pool will never have fewer than this number of threads. It will grow as needed up to the maximum value set by the `nifi.cluster.node.protocol.max.threads`
    property.
 ** `nifi.cluster.node.protocol.max.threads` - The maximum number of threads that should be used to communicate with other nodes in the cluster. This property
-	defaults to `50`. A thread pool is used for replication requests to all nodes, and the thread pool will have a "core" size that is configured by the
```

Comment:
Could you revert all the unnecessary changes on the documentation?

Reply:
Oof, yeah.  I would blame my tools if I could.  Thanks for the catch.